docs/DeveloperGuide.md

```diff
@@ -36,3 +33,115 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+
+<!-- Zhan hong working on this section-->
+# Review Feature
+**API:** [Review.java](https://github.com/AY2223S2-CS2113-W12-4/tp/blob/master/src/main/java/Review.java)
+
+Here's a partial class diagram of the `Review` component:
+
+![](/PUMLFiles/Review/review1.png)
```

Comment:
I see that there are `C` or `I` located at the left side of the class name. perhaps finding a way to remove those indicators would be nice as those indicators are not part of standard UML indicators.